src/lib/OpenAI/Auth/BaseManifestAuth.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.OpenApi.ApiManifest.OpenAI.Auth;
+using System.Text.Json;
+
+namespace Microsoft.OpenApi.ApiManifest.OpenAI;
```

Comment:
why isn't this type in the auth namespace? if it's generic (not OpenAI specific) should it be outside of the OpenAI namespace?

Reply:
The base type should be under auth namespace since it is OpenAI specific. I'll move it under auth.